com.varnamproject.Varnam.json

```diff
@@ -0,0 +1,171 @@
+                    "sha256": "67045236dcfd6816dc439556d009594abf643e5eb48992e36beac09c2ca659b8"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/f8/e8/b30ae23b45f69aa3f024b46064c0ac8e5fcb4f22ace0dca8d6f9c8bbe5e7/greenlet-0.4.15.tar.gz",
+                    "sha256": "9416443e219356e3c31f1f918a91badf2e37acf297e2fa13d24d1cc2380f8fbc"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/98/d2/6fa19239ff1ab072af40ebf339acd91fb97f34617c2ee625b8e34bf42393/gevent-websocket-0.10.1.tar.gz",
+                    "sha256": "7eaef32968290c9121f7c35b973e2cc302ffb076d018c9068d2f5ca8b2d85fb0"
+                }
+            ]
+        },
+        {
+            "name": "varnam-import",
+            "buildsystem": "simple",
+            "build-options" : {
+                "env": [
+                    "VARNAM_SUGGESTIONS_DIR=suggestions"
+                ]
+            },
+            "build-commands": [
+                "install -d /app/share/com.varnamproject.Varnam/suggestions",
+                "tar -xf ml.full.tar.gz",
+                "varnamc -s ml --import-learnings-from .",
+                "rm $VARNAM_SUGGESTIONS_DIR/*.learnings-*",
+                "mv $VARNAM_SUGGESTIONS_DIR/* /app/share/com.varnamproject.Varnam/suggestions"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "http://mirror.rackdc.com/savannah/varnamproject/words/ml.full.tar.gz",
+                    "sha256": "b4d7208066a1395b033be5223ffbbc7759aabec6a7a2c60d9297cdd71c56dffd"
+                }
+            ]
+        },
+        {
+            "name": "varnam-wrapper",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST}"
+            ],
+            "post-install": [
+                "chmod +x varnam-wrapper.sh",
+                "cp varnam-wrapper.sh /app/bin/varnam-wrapper",
+                "install -Dm644 com.varnamproject.Varnam.appdata.xml /app/share/appdata/com.varnamproject.Varnam.appdata.xml",
+                "install -Dm644 com.varnamproject.Varnam.desktop /app/share/applications/com.varnamproject.Varnam.desktop",
+                "install -Dm644 varnam.png /app/share/icons/hicolor/48x48/apps/com.varnamproject.Varnam.png"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/subins2000/varnam.git",
+                    "branch": "master"
+                },
+                {
+                    "type": "file",
+                    "path": "varnam-wrapper.sh"
+                },
+                {
+                    "type": "file",
+                    "path": "com.varnamproject.Varnam.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html